diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -176,6 +176,10 @@ open class DiktatJavaExecTaskBase @Inject constructor(
             }
             .absolutePath
     }
+
+    companion object {
+        private val log = LoggerFactory.getLogger(DiktatJavaExecTaskBase::class.java)
```

Comment:
We should be using `project.logger` in gradle plugin, I think we imported our logger here by mistake. Could you please change it's usages in this file? Then this logger will be not needed.

Reply:
yeah, sure